README.md

```diff
@@ -1,12 +1,30 @@
 # node-rhsplib
 
-This package enables Node.js applications to control devices that speak the REV Hub Serial Protocol (such as the REV Robotics Expansion Hub).
+This project enables Node.js applications to control devices that speak the REV Hub Serial Protocol (such as the REV Robotics Expansion Hub).
+
+## package structure
+
+This project uses [lerna](https://lerna.js.org) for
+multi-package builds.
+
+We have several packages currently:
+
+1. rhsplib: Raw interaction with RHSPLib.
+1. expansion-hub: High-level interaction with a REV expansion hub.
```

Comment:
Markdown spec allows us to number all items '1.', and compliant viewers should show the proper numbers. This allows us to easily add an item to the middle of the list without having to worry about committing a large number of changes when later items have the number change.

Reply:
Hmm, that's a clever way to abuse the spec. However, it hurts the readability of the plain text version, which I am not a fan of. I think I'd rather have them be the correct numbers, even if it means that more changes are needed when editing the list. A good markdown editor should let you auto-fix that.